docs/diagrams/add/AddCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":RealodexParser" as RealodexParser LOGIC_COLOR
+participant ":AddCommandParser" as AddCommandParser LOGIC_COLOR
+participant "r:AddCommand" as AddCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("add n/John Doe a/6 College Ave West ..")
+activate LogicManager
+
+LogicManager -> RealodexParser : parseCommand("add n/John Doe a/6 College Ave West ..")
```

Comment:
I think no need to put in the argument, you can add it outside of the diagram to increase readability